src/cpp/cpp_typecheck.cpp

```diff
@@ -323,3 +328,13 @@ void cpp_typecheckt::clean_up()
     }
   }
 }
+
+bool cpp_typecheckt::contains_cpp_name(const exprt &expr)
+{
+  if(expr.id()==ID_cpp_name)
+    return true;
+  forall_operands(it, expr)
```

Comment:
Nit: can you use a range based for loop here? Also since for loop extends over multiple lines please wrap in braces

Reply:
Done in [PR #6](https://github.com/tautschnig/cbmc/pull/6)